packages/compiler/core/util.ts

```diff
@@ -48,6 +48,10 @@ export function deepClone<T>(value: T): T {
   return value;
 }
 
+export async function resolveRealPath(host: CompilerHost, path: string) {
+  return resolvePath(await host.realpath(path));
```

Comment:
```suggestion
  return normalizePath(await host.realpath(path));
```
Resolve path calls out to `normalizePath` + some resolution if multiple args